vet.sh

```diff
@@ -163,4 +163,36 @@ naming.Watcher
 resolver.Backend
 resolver.GRPCLB' "${SC_OUT}"
 
+# - special golint on package comments.
+lint_package_comment_per_package() {
+  # Number of files in this go package.
+  fileCount=$(go list -f '{{len .GoFiles}}' $1)
+  if [ ${fileCount} -eq 0 ]; then
+    return 0
+  fi
+  # Number of package errors generated by golint.
+  lintPackageCommentErrorsCount=$(golint --min_confidence 0 $1 | grep -c "should have a package comment" || true)
+  # golint complains about every file that's missing the package comment. If the
+  # number of files for this package is greater than the number of errors, there's
+  # at least one file with package comment, good. Otherwise, fail.
+  if [ ${fileCount} -le ${lintPackageCommentErrorsCount} ]; then
+    echo "Package $1 (with ${fileCount} files) is missing package comment"
+    return 1
+  fi
+}
+lint_package_comment() {
+  echo "+ running golint for package comment"
```

Comment:
You don't really need this line at all:

```
+lint_package_comment
+echo '+ running golint for package comment'
+ running golint for package comment
```

Just `lint_package_comment` and the errors (if any are produced) are sufficient.

Reply:
Removed